code/obj/item/organs/liver.dm

```diff
@@ -32,3 +32,30 @@
 	robotic = 1
 	edible = 0
 	mats = 6
+	var/overloading = 0
+
+	emag_act(mob/user, obj/item/card/emag/E)
+		. = ..()
+		organ_abilities = list(/datum/targetable/organAbility/liverdetox)
+
+	demag(mob/user)
+		..()
+		organ_abilities = initial(organ_abilities)
+
+	on_life(var/mult = 1)
+		if(!..())
+			return 0
+		if(overloading)
```

Comment:
Maybe livers should take damage while overloading even if they have no ethanol to convert? Some pained messages perhaps to let the player know? Unless this was an intentional decision, I guess I can see it going either way. 
But "overloading" sounds to me like it should be dangerous whenever it's being overloaded.